app/frontend/src/features/auth/signup/CompleteSignupForm.tsx

```diff
@@ -204,8 +205,14 @@ export default function CompleteSignupForm() {
             <Datepicker
               className={authClasses.formField}
               control={control}
-              error={!!errors?.birthdate?.message}
-              helperText={errors?.birthdate?.message ?? " "}
+              error={
+                //@ts-ignore Dayjs type breaks this
```

Comment:
What errors do you get with this? Would have thought it'd be ok with this since you are converting the value to boolean with `!!`

Reply:
From memory, something like `DeepMap<Dayjs, FieldError> does not contain member 'message'`